components/swatchgroup/stories/swatchgroup.stories.js

```diff
@@ -1,16 +1,41 @@
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
+import { Sizes } from "@spectrum-css/preview/decorators";
 import { default as Swatch } from "@spectrum-css/swatch/stories/swatch.stories.js";
 import { version } from "../package.json";
-import { SwatchGroup } from "./swatchgroup.test.js";
+import { SwatchgroupGroup } from "./swatchgroup.test.js";
+import { Template, RoundingTemplate } from "./template";
 
 /**
- * The swatch group component is a collection of swatches.
+ * The swatch group component is a collection of [swatches](?path=/docs/components-swatch--docs).
+ * 
+ * ## Usage notes
+ * 
+ * ### Corner rounding in swatch groups
+ * 
+ * A corner rounding of “none” (`.spectrum-Swatch--roundingNone` class) should be used in a swatch group in order to help minimize the Hermann grid illusion that happens at the intersections of the white space within the group.
+ * 
+ * The only exception is when a swatch group only takes up a single row. In that case, use any of the rounding options.
+ * 
+ * ### Border only for low-contrast swatches
+ * 
+ * When swatches within a swatch group have low contrast (below 3:1 contrast with the background), they have a less prominent border compared to the swatch component when used by itself, and should have the `.spectrum-Swatch--lightBorder` class. Low contrast color swatches have a border of `gray-900` at 20% opacity. This reduces the likelihood of the UI interfering with color perception and comparisons.
+ * 
+ * Otherwise, swatches within a swatch group should have the `.spectrum-Swatch--noBorder` class. 
+ * 
+ * ### Density
+ * 
+ * Swatch groups come in 3 densities: regular (default), compact, and spacious. Compact and spacious densities retain the same swatch size as regular density, but have less or more padding between each swatch, respectively.
  */
 export default {
 	title: "Swatch group",
 	component: "SwatchGroup",
 	argTypes: {
 		...Swatch.argTypes,
+		swatchColor: { table: { disable: true } },
+		isRectangle: { table: { disable: true } },
+		imageUrl: { table: { disable: true } },
+		isMixedValue: { table: { disable: true } },
+		gradient: { table: { disable: true } },
```

Comment:
These were all in the controls for swatch group (since we're spreading the `Swatch` args, and I had to add a bunch to swatch), but I didn't think they were necessary. Let me know if I should keep them in the controls table.